soccer_rlcontrol/README.md

```diff
@@ -0,0 +1,72 @@
+# Installation
+Make sure you are in the same directory as the README
+```shell script
+sudo apt-get install python3-tk
+python3 -m pip install --user virtualenv
+virtualenv -p python3 venv
+. venv/bin/activate
+pip install -r requirements.txt
+```
+Then, install the gym-soccerbot package
+```shell script
+cd gym-soccerbot
+pip install -e .
```

Comment:
```suggestion
pip3 install -e .
```

Reply:
Did you encounter any issues that were resolved by this change? Was the virtualenv activated when those issues appeared?